apps/worker/src/app/workflow/usecases/send-message/digest/digest.usecase.ts

```diff
@@ -58,7 +58,10 @@ export class Digest extends SendMessageType {
   private async getEvents(command: SendMessageCommand) {
     const currentJob = await this.jobRepository.findOne({ _environmentId: command.environmentId, _id: command.jobId });
 
-    if (currentJob?.digest?.type === DigestTypeEnum.BACKOFF || currentJob?.digest?.backoff) {
+    if (
+      currentJob?.digest?.type === DigestTypeEnum.BACKOFF ||
+      (currentJob?.digest && 'backoff' in currentJob?.digest)
```

Comment:
Will not this be true even if backoff is false?

Reply:
🎖️ good catch